### PR TITLE
glossary: Add more abbreviations

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

Closes #19212 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/19213
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: glossary.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ aliases: ['/docs-cn/dev/glossary/']
 
 # 术语表
 
+本术语表提供了 TiDB 相关的关键术语定义。
```
Comment (qiancai) [inline glossary.md:9]:
```suggestion
本术语表提供了 TiDB 中的关键术语定义。
```

File: glossary.md [hunk 2]
```diff
@@ -19,18 +27,11 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
...
-* 架构中的 BR 功能组件。
+### Backup & Restore (BR)
 
-名词 **br** 一般用来指代 br CLI 工具。
+**Backup & Restore** 或 **BR** 指代 [TiDB 备份恢复功能](/br/backup-and-restore-overview.md)。
 
-### Batch Create Table
-
-批量建表 (Batch Create Table) 是在 TiDB v6.0.0 中引入的新功能，此功能默认开启。当需要恢复的数据中带有大量的表（约 50000 张）时，批量建表功能显著提升数据恢复的速度。详情参见[批量建表](/br/br-batch-create-table.md)。
+`br` 指代进行 TiDB 备份或恢复时使用的 [br 命令行工具](/br/use-br-command-line-tool.md)。
```
Comment (hfxsd) [inline glossary.md:35]: Batch Create Table 为啥删掉了？英文是有的 https://github.com/pingcap/docs/pull/19213/files#diff-69341bc40c73d6527963e61565410d7beb753362f23a79c130cf1d532e6587ddR43

File: glossary.md [hunk 3]
```diff
@@ -50,35 +51,105 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
 
 Coalesce Partition 是一种减少 Hash 分区表或 Key 分区表中分区数量的方法。详情参见[管理 Hash 分区和 Key 分区](/partitioned-table.md#管理-hash-分区和-key-分区)。
 
+### Column Family (CF)
+
+在 RocksDB 和 TiKV 中，Column Family (CF，列族) 表示数据库中键值对的逻辑分组。
+
+### 公共表表达式 (CTE)
+
+公共表表达式（Common Table Expression, CTE）用于定义一个临时结果集，能够在 SQL 语句中通过 [`WITH`](/sql-statements/sql-statement-with.md) 子句多次引用。更多信息，请参见[公共表表达式](/develop/dev-guide-use-common-table-expression.md)。
```
Comment (hfxsd) [inline glossary.md:60]:
```suggestion
公共表表达式 (Common Table Expression, CTE) 用于定义一个临时结果集，能够在 SQL 语句中通过 [`WITH`](/sql-statements/sql-statement-with.md) 子句多次引用。更多信息，请参见[公共表表达式](/develop/dev-guide-use-common-table-expression.md)。
```

Comment (hfxsd) [inline glossary.md:60]:
```suggestion
公共表表达式 (Common Table Expression, CTE) 用于定义一个临时结果集，能够在 SQL 语句中通过 [`WITH`](/sql-statements/sql-statement-with.md) 子句多次引用。更多信息，请参见[公共表表达式](/develop/dev-guide-use-common-table-expression.md)。
```

File: glossary.md [hunk 4]
```diff
@@ -50,35 +51,105 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
...
+
 ### Continuous Profiling
 
 持续性能分析 (Continuous Profiling) 是从 TiDB v5.3 起引入的一种从系统调用层面解读资源开销的方法。引入该方法后，TiDB 可提供数据库源码级性能观测，通过火焰图的形式帮助研发、运维人员定位性能问题的根因。详情参见 [TiDB Dashboard 实例性能分析 - 持续分析页面](/dashboard/continuous-profiling.md)。
 
 ## D
 
+### Data Definition Language (DDL)
+
+数据定义语言（Data Definition Language, DDL）是 SQL 标准的一部分，用于创建、修改和删除表及其他对象。更多信息，请参见 [DDL 语句的执行原理及最佳实践](/ddl-introduction.md)。
```
Comment (hfxsd) [inline glossary.md:70]:
```suggestion
数据定义语言 (Data Definition Language, DDL) 是 SQL 标准的一部分，用于创建、修改和删除表及其他对象。更多信息，请参见 [DDL 语句的执行原理及最佳实践](/ddl-introduction.md)。
```

File: glossary.md [hunk 5]
```diff
@@ -50,35 +51,105 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
...
+
+### Data Migration (DM)
+
+Data Migration (DM) 是 TiDB 提供的一款数据迁移工具，用于将数据从 MySQL 兼容的数据库迁移到 TiDB。DM 会从 MySQL 兼容的数据库实例读取数据，然后将其应用到 TiDB 目标实例中。更多信息，请参见 [TiDB Data Migration 简介](/dm/dm-overview.md)。
+
+### Data Modification Language (DML)
+
+数据操作语言（Data Modification Language, DML）是 SQL 标准的一部分，用于插入、更新和删除表中的行数据。
```
Comment (hfxsd) [inline glossary.md:78]:
```suggestion
数据操作语言 (Data Modification Language, DML) 是 SQL 标准的一部分，用于插入、更新和删除表中的行数据。
```

File: glossary.md [hunk 6]
```diff
@@ -50,35 +51,105 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
...
+
+### 开发里程碑版本 (DMR)
+
+TiDB 会在开发里程碑版本（Development Milestone Release, DMR）中引入新的功能，但 DMR 不提供长期支持。更多信息，请参见 [TiDB 版本规则](/releases/versioning.md)。
+
+### 容灾 (DR)
+
+容灾（Disaster Recovery, DR）是在未来灾难发生时恢复数据和服务的解决方案。TiDB 提供了多种容灾方案，例如备份和复制数据到备用集群。更多信息，请参见 [TiDB 容灾方案概述](/dr-solution-introduction.md)。
```
Comment (hfxsd) [inline glossary.md:86]:
```suggestion
容灾 (Disaster Recovery, DR) 是在未来灾难发生时恢复数据和服务的解决方案。TiDB 提供了多种容灾方案，例如备份和复制数据到备用集群。更多信息，请参见 [TiDB 容灾方案概述](/dr-solution-introduction.md)。
```

File: glossary.md [hunk 7]
```diff
@@ -50,35 +51,105 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
...
+
+### 分布式执行框架 (DXF)
+
+分布式执行框架（Distributed eXecution Framework, DXF）允许 TiDB 在处理特定任务（例如创建索引或导入数据）时对这些任务进行统一调度和分布式执行。该框架旨在高效利用集群资源执行任务，控制资源使用，以减少对核心业务事务的影响。更多信息，请参见 [TiDB 分布式执行框架](/tidb-distributed-execution-framework.md)。
```
Comment (hfxsd) [inline glossary.md:90]:
```suggestion
分布式执行框架 (Distributed eXecution Framework, DXF) 允许 TiDB 在处理特定任务（例如创建索引或导入数据）时对这些任务进行统一调度和分布式执行。该框架旨在高效利用集群资源执行任务，控制资源使用，以减少对核心业务事务的影响。更多信息，请参见 [TiDB 分布式执行框架](/tidb-distributed-execution-framework.md)。
```

File: glossary.md [hunk 8]
```diff
@@ -50,35 +51,105 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
...
+
 ### Dynamic Pruning
 
 动态裁剪 (Dynamic Pruning) 是 TiDB 访问分区表的两种模式之一。在动态裁剪模式下，TiDB 的每个算子都支持直接访问多个分区，省略 Union 操作，提高执行效率，还避免了 Union 并发管理的问题。
 
+## G
+
+### Garbage Collection (GC)
+
+垃圾回收（Garbage Collection, GC）指清理不再需要的旧数据以释放资源的过程。关于 TiKV 垃圾回收过程的详情，请参见 [垃圾回收概述](/garbage-collection-overview.md)。
```
Comment (hfxsd) [inline glossary.md:100]:
```suggestion
垃圾回收（Garbage Collection, GC）指清理不再需要的旧数据以释放资源的过程。关于 TiKV 垃圾回收过程的详情，请参见[垃圾回收概述](/garbage-collection-overview.md)。
```

File: glossary.md [hunk 9]
```diff
@@ -50,35 +51,105 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
...
+
+### General Availability (GA)
+
+一个功能 GA (General Availability) 意味着该功能已进行充分测试并可在生产环境中使用。根据每个功能的开发情况不同，TiDB 中的新功能可能会在 [开发里程碑版本 (DMR)](#development-milestone-release-dmr) 中 GA，也可能会在 [长期支持版本 (LTS)](#long-term-support-lts) 中 GA 。由于 TiDB 不提供基于 DMR 的补丁版本，在生产环境中建议使用 LTS 版本。
+
+### Global Transaction Identifiers (GTIDs)
+
+全局事务标识符（Global Transaction Identifiers, GTIDs）是在 MySQL 二进制日志中跟踪已复制事务的唯一标识符。[Data Migration (DM)](/dm/dm-overview.md) 在迁移数据时会使用这些标识符确保复制的一致性。
```
Comment (hfxsd) [inline glossary.md:108]:
```suggestion
全局事务标识符 (Global Transaction Identifiers, GTIDs) 是在 MySQL 二进制日志中跟踪已复制事务的唯一标识符。[Data Migration (DM)](/dm/dm-overview.md) 在迁移数据时会使用这些标识符确保复制的一致性。
```

File: glossary.md [hunk 10]
```diff
@@ -50,35 +51,105 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
...
+
+## H
+
+### Hybrid Transactional and Analytical Processing (HTAP)
+
+混合型在线事务与在线分析处理 （Hybrid Transactional and Analytical Processing, HTAP）功能支持在同一数据库中同时处理 OLTP（联机事务处理）和 OLAP（联机分析处理）工作负载。在 TiDB 中，HTAP 是通过使用 TiKV 进行行存以及使用进行 TiFlash 进行列存来实现的。更多信息，请参见 [Gartner 网站上的 HTAP 定义](https://www.gartner.com/en/information-technology/glossary/htap-enabling-memory-computing-technologies)。
```
Comment (hfxsd) [inline glossary.md:114]:
```suggestion
混合型在线事务与在线分析处理 (Hybrid Transactional and Analytical Processing, HTAP) 功能支持在同一数据库中同时处理 OLTP（联机事务处理）和 OLAP（联机分析处理）工作负载。在 TiDB 中，HTAP 是通过使用 TiKV 进行行存以及使用进行 TiFlash 进行列存来实现的。更多信息，请参见 [Gartner 网站上的 HTAP 定义](https://www.gartner.com/en/information-technology/glossary/htap-enabling-memory-computing-technologies)。
```

File: glossary.md [hunk 11]
```diff
@@ -50,35 +51,105 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
...
 
 索引合并 (Index Merge) 是在 TiDB v4.0 版本中作为实验特性引入的一种查询执行方式的优化，可以大幅提高查询在扫描多列数据时条件过滤的效率。自 v5.4 版本起，Index Merge 成为正式功能，详情参见[用 EXPLAIN 查看索引合并的 SQL 执行计划](/explain-index-merge.md)。
 
-### In-Memory Pessimistic Lock
+## K
 
-内存悲观锁 (In-Memory Pessimistic Lock) 是在 TiDB v6.0.0 中引入的新功能。开启内存悲观锁功能后，悲观锁通常只会被存储在 Region leader 的内存中，而不会将锁持久化到磁盘，也不会通过 Raft 协议将锁同步到其他副本，因此可以大大降低悲观事务加锁的开销，提升悲观事务的吞吐并降低延迟。
+### Key Management Service (KMS)
+
+密钥管理服务（Key Management Service, KMS）提供了一种存储和检索密钥的安全方式。常见的 KMS 包括 AWS KMS、Google Cloud KMS 和 HashiCorp Vault。TiDB 中的多个组件都支持通过 KMS 管理用于存储加密和相关服务的密钥。
```
Comment (hfxsd) [inline glossary.md:130]:
```suggestion
密钥管理服务 (Key Management Service, KMS) 提供了一种存储和检索密钥的安全方式。常见的 KMS 包括 AWS KMS、Google Cloud KMS 和 HashiCorp Vault。TiDB 中的多个组件都支持通过 KMS 管理用于存储加密和相关服务的密钥。
```

File: glossary.md [hunk 12]
```diff
@@ -117,16 +200,28 @@ Operator Step 是 Operator 执行过程的一个步骤，一个 Operator 常常
...
+
 ### Predicate columns
 
 执行 SQL 语句时，优化器在大多数情况下只会用到部分列（例如，`WHERE`、`JOIN`、`ORDER BY`、`GROUP BY` 子句中出现的列）的统计信息，这些用到的列称为 `PREDICATE COLUMNS`。详情参见[收集部分列的统计信息](/statistics.md#收集部分列的统计信息)。
 
 ## Q
 
+### Queries Per Second (QPS)
+
+每秒查询数(Queries Per Second, QPS) 指的是数据库服务每秒处理的查询数量。它是衡量数据库吞吐量的重要性能指标。
```
Comment (qiancai) [inline glossary.md:223]:
```suggestion
每秒查询数 (Queries Per Second, QPS) 指的是数据库服务每秒处理的查询数量。它是衡量数据库吞吐量的重要性能指标。
```

Comment (hfxsd) [inline glossary.md]:
```suggestion
每秒查询数 (Queries Per Second, QPS) 指的是数据库服务每秒处理的查询数量。它是衡量数据库吞吐量的重要性能指标。
```

File: glossary.md [hunk 13]
```diff
@@ -19,18 +27,11 @@ ACID 是指数据库管理系统在写入或更新资料的过程中，为保证
...
-* 架构中的 BR 功能组件。
+### Backup & Restore (BR)
 
-名词 **br** 一般用来指代 br CLI 工具。
+**Backup & Restore** 或 **BR** 指代 [TiDB 备份恢复功能](/br/backup-and-restore-overview.md)。
 
-### Batch Create Table
-
-批量建表 (Batch Create Table) 是在 TiDB v6.0.0 中引入的新功能，此功能默认开启。当需要恢复的数据中带有大量的表（约 50000 张）时，批量建表功能显著提升数据恢复的速度。详情参见[批量建表](/br/br-batch-create-table.md)。
+`br` 指代进行 TiDB 备份或恢复时使用的 [br 命令行工具](/br/use-br-command-line-tool.md)。
```
Comment (qiancai) [inline glossary.md:34]:
```suggestion
`br` 指代进行 TiDB 备份或恢复时使用的 [br 命令行工具](/br/use-br-command-line-tool.md)。

### Batch Create Table

批量建表 (Batch Create Table) 是在 TiDB v6.0.0 中引入的新功能，此功能默认开启。当需要恢复的数据中带有大量的表（约 50000 张）时，批量建表功能显著提升数据恢复的速度。详情参见[批量建表](/br/br-batch-create-table.md)。
```

File: glossary.md (hunk 2 above)
Comment (qiancai) [reply glossary.md:35]: added in [`6a751ae` (#19029)](https://github.com/pingcap/docs-cn/pull/19029/commits/6a751ae33564ab4d7e6c801101e916ea35b0f8b3)

File: glossary.md [hunk 14]
```diff
@@ -137,15 +236,23 @@ Pending 和 Down 是 Peer 可能出现的两种特殊状态。其中 Pending 表
 
 一种内置的持久化存储引擎，有着日志结构的设计，为 TiKV 提供 multi-Raft 日志存储。从 v5.4 起，TiDB 支持使用 Raft Engine 作为 TiKV 的日志存储引擎。详情参见 [Raft Engine](/tikv-configuration-file.md#raft-engine)。
 
+### Region Split
+
+TiKV 集群中的 Region 不是一开始就划分好的，而是随着数据写入逐渐分裂生成的，分裂的过程被称为 Region Split。
+
```
Comment (hfxsd) [inline glossary.md:242]: 描述和英文有差异。中文似乎更详细，已经同步修改了英文 https://github.com/pingcap/docs/pull/19213/files#r1841400794

File: glossary.md [hunk 15]
```diff
@@ -137,15 +236,23 @@ Pending 和 Down 是 Peer 可能出现的两种特殊状态。其中 Pending 表
...
+其机制是集群初始化时构建一个初始 Region 覆盖整个 key space，随后在运行过程中每当 Region 数据达到一定量之后就通过 Split 产生新的 Region。
```
Comment (hfxsd) [inline glossary.md:243]: 和英文描述不同，every time the size of the Region or the number of keys has reached a threshold.
```suggestion
其机制是集群初始化时构建一个初始 Region 覆盖整个 key space，随后在运行过程中每当 Region 大小或 Key 数量达到阈值之后就通过 Split 产生新的 Region。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**